axelrod/strategies/axelrod_first.py

```diff
@@ -487,3 +489,69 @@ class UnnamedStrategy(Player):
     def strategy(opponent: Player) -> Action:
         r = random.uniform(3, 7) / 10
         return random_choice(r)
+
+@FinalTransformer((D, D), name_prefix=None)
+class SteinAndRapoport(Player):
+    """
+        A player who plays according to statistic methods.
+        Begins by playing C for the first four (4) rounds , then it plays
+        tit for tat and at the last 2 round it Defects. Every 15 turns it
+        run a chi-squared test to check whether the opponent behaves randomly
+        or not . In case the opponent behaves randomly then Stein_and_Rapoport
+        Defects untill the next 15 round (where we check again), otherwise he
+        still plays TitForTat.
+    """
+
+    name = 'Stein and Rapoport'
+    classifier = {
+        'memory_depth': 15,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def __init__(self, alpha: float=0.05) -> None:
+        """
+        Parameters
+        ----------
+        alpha, float
+            The significant level of pvalue from chi-squared test
+            0.05 by default according to literature
+        """
+        super().__init__()
+        self.alpha = alpha
+        if (self.alpha > 1) or (self.alpha < 0):
+            self.alpha = 0.05
+
+    def strategy(self , opponent: Player , chi_tests = [0]) -> Action:
+        # chi-tests == 0 then we play TitForTat
+        round_number = len(self.history) + 1
+
+        # First 4 moves
+        if round_number < 5 :
```

Comment:
`round_number < 5 :` should be `round_number < 5:` (no space between `5` and `:`)